res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -858,6 +859,7 @@ NumarkMixtrack3.deck = function(decknum) {
     // NMTP3 is a "Model A" controller for scratching, it centers on 0.
     // See http://www.mixxx.org/wiki/doku.php/midi_scripting#scratching
     // and see "Jogger" object constructor
+    this.beatJumpSize = 1;
```

Comment:
In the future, this will be available as a ControlObject that will also be shown on skins so controller scripts won't have to keep track of this. I might implement that for Mixxx 2.2. There is a [Launchpad ticket](https://bugs.launchpad.net/mixxx/+bug/1429331) tracking it.